data/json/snippets/mist.json

```
@@ -0,0 +1,53 @@
+[
+  {
+    "type": "snippet",
+    "category": "mist_increase_intensity",
+    "text": [ "The mist thickens.", "The mist feels slightly stronger and more alive." ]
+  },
+  {
+    "type": "snippet",
+    "category": "mist_arrives",
+    "text": [
+      "You blink and there is dense mist as far as you can see, which is much less than it used to be.",
+      "In an instant a thick mist descends upon the world, far too fast to be natural.  Or are you finally losing it?."
```

```suggestion
      "In an instant a thick mist descends upon the world, far too fast to be natural."
```
I think the 'losing it' bit inserts too much of the character's thoughts